### DISCUSSION TITLE
A user community meet-up in the Bay Area, Tue. June 27

### DISCUSSION THREAD

**Discussion by erez-tom** [google-deepmind/mujoco#944] [General]
Hello,

Members of the MuJoCo development team will be in California next week, and we want to invite you all to a public meet-up, at or near Mountain View, on Tue., June 27, 6-8pm, at the [State Street Market](https://www.statestreetmarket.com/) in Los Altos. 

[Please RSVP using this form](https://rsvp.withgoogle.com/events/mujoco-bay-area-meetup-june-2023), so we know how many people to expect (and we can update you on the exact location).  We are not currently planning to offer any dinner or drinks, but those should be amply available for purchase (e.g., from the different stalls of the market).

**Comment by kevinzakka** [comment]
Exciting!!

**Reply by erez-tom** [reply]
I've updated the text above -- we're confirmed for the State Street Market in Los Altos.  You're still welcome to join, but please add your name through the RSVP form above.
